weather_app/weather/models.py

```diff
@@ -10,3 +10,110 @@ def __str__(self):
+    nombre = models.CharField(max_length=25)
+    apellido = models.CharField(max_length=25)
+
+class Genero(models.Model):
+    cantante = models.ForeignKey(Cantante, on_delete=models.CASCADE)
+    generom = models.CharField(max_length=25)
+    
+
+class Estudiante(models.Model):
+    primer_nombre = models.CharField(max_length=25)
+    segundo_nombre = models.CharField(max_length=25)
+    primer_apellido = models.CharField(max_length=25)
+    segundo_apellido = models.CharField(max_length=25)
+
+class Materias(models.Model):
+    estudiante =models.ForeignKey(Estudiante, on_delete=models.PROTECT)
+    grado =models.CharField(max_length=25)
+    materias_reprobadas = models.IntegerField()
+
+class PersonasViaje(models.Model):
+    nombre = models.CharField(max_length=25)
+    apellido = models.CharField(max_length=25)
+    destino = models.CharField(max_length=25)
+    plata_abonada = models.IntegerField()
+
+
+class InscripcionJuegosMesa(models.Model):
+    nombre = models.CharField(max_length=25)
+    apellido = models.CharField(max_length=25)
+    documento = models.IntegerField()
+
+class JuegosDeMesa(models.Model):
+    jugadorm = models.ForeignKey(InscripcionJuegosMesa, on_delete=models.CASCADE)
+    juegos_disponibles = models.CharField(max_length=40)
+
+
+class AnimalesZoologico(models.Model): 
+    nombre = models.CharField(max_length=25)
+    lugar_de_origen = models.CharField(max_length=25)
+    cantidad = models.IntegerField()
+    fecha_ingreso = models.DateField()
+
+
+class Registro(models.Model):
+    nombre_completo = models.CharField(max_length=25)
+    documento = models.IntegerField()
+    hora_de_llegada = models.TimeField()
+    fecha = models.DateField()
+
+class ObjetosEncontrados(models.Model):
+    nombre_objeto = models.CharField(max_length=25)
+
+class DescripcionDeObjetosEncontrados(models.Model):
+    objeto = models.ForeignKey(ObjetosEncontrados, on_delete=models.PROTECT)
+    descripcion_estado_de_objeto = models.TextField(max_length=200)
+    
+
+class Celulares(models.Model):
+    marca_celular =models.CharField(max_length=25)
+
+class Precios(models.Model):
+    celular =models.ForeignKey(Celulares, on_delete=models.CASCADE)
+    precios_sugeridos= models.URLField()
+
+class Ventas(models.Model):
+    producto =models.CharField(max_length=25)
+    cantidad =models.IntegerField()
+    hora_fecha=models.DateTimeField()
+
+class VentaVideojuegos(models.Model):
+    nombre_juego=models.CharField(max_length=45)
+    precio =models.FloatField()
+
+class VentaViajes(models.Model):
+    destino = models.CharField(max_length=25)
+    valor_viaje =models.FloatField()
+    descuento = models.BooleanField()
+    
+class RegistroUsuario(models.Model):
+    nombre_usuario=models.CharField(max_length=25)
+    documento = models.IntegerField()
+
+class Registro(models.Model):
+    usuario = models.ForeignKey(RegistroUsuario, on_delete=models.PROTECT)
+    correo = models.EmailField()
+
+
+class FechaCumpleaños(models.Model):
```

Comment:
```suggestion
class FechaCumpleannos(models.Model):
```
Python es un poquito delicado con el tema de caracteres especiales, como en inglés no existe la ñ es mejor que lo dejes con nn o n

Reply:
listo ya lo cambie